app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/ArticleView.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="h-full w-full flex flex-col flex-1 overflow-hidden">
```

Comment:
@nithindavid Is it possible to set a max height? Now it looks like this on bigger resolution screens. And can we reduce the gap between the editor and the article container (Only if the woot-editor is empty)?
![image](https://github.com/chatwoot/chatwoot/assets/64252451/7bfc5ff2-7f7a-43b7-8f1c-93056df6fe89)

Reply:
Sounds good. I'll set it 👍